elements/pfe-primary-detail/pfe-primary-detail.ts

```diff
@@ -94,21 +94,16 @@ export class PfePrimaryDetail extends LitElement {
   /** The min-width of the _component_ where it has a desktop layout */
   @property({ attribute: 'breakpoint-width', type: Number, reflect: true }) breakpointWidth = 800;
 
-  /** Read only: Displays the id of an open 'detailNav' element */
   @observed
   @property({ type: String, reflect: true }) active: string|null = null;
 
-  /** Read only: Displays what breakpoint is currently being used */
-  @property({ type: String, reflect: true }) breakpoint?: 'compact'|'desktop';
-
-  /** Used to set text of back button **/
   @property({ type: String }) expandedSectionTitle?: string;
```

Comment:
```suggestion
  /** Used to set text of back button **/
  @property({ type: String }) expandedSectionTitle?: string;
```

we should keep the docs here for now. later on we can make this private if we like